### DISCUSSION TITLE
Scroll Layout or menu for dedicated pages

### DISCUSSION THREAD

**Discussion by Papricasa** [janraasch/hugo-scroll#183] [Ideas]
Hey :) 
I want to clean up my website and put some stuff in subpages. The subpages also should have a menu or maybe even the same layout then the starter-scroll-site. Is there a shortcode to do so?
Or how else can I do this? 

[And th@nks for this nice Template] 
Papricasa

**Comment by zjedi** [comment]
Hi @Papricasa, yes I agree with you, but unfortunately the subpages have not been developed properly yet. There is #116 filed for that. Feel to do so. I might give some guidance, but currently hot having capacity to develop it myself.
